fiber/online/R3BFiberMAPMTOnlineSpectra.cxx

```diff
@@ -575,15 +570,17 @@ void R3BFiberMAPMTOnlineSpectra::Exec(Option_t* /*option*/)
             if (!hit)
                 continue;
 
+            if (!header)
+                continue;
+
             iFib = hit->GetFiberId(); // 1..
 
             // times
             tUp = hit->GetTopTime_ns();
             tDown = hit->GetBottomTime_ns();
             dtime = tUp - tDown;
 
-            // (tUp+tDown)/2:
-            tfib = hit->GetTime();
+            tfib = hit->GetTime_ns() - header->GetTStart();
```

Comment:
hit->GetTime() already provides the time difference wrt to LOS, maybe something is wrong in the cal2hit.